### ISSUE TITLE
[Feature Request] Keyboard shortcut to scroll while reading

### ISSUE THREAD

**Issue by heliostatic** [FreshRSS/FreshRSS#1842]
The keyboard shortcuts are wonderful! Would it be possible to add a shortcut for scrolling the page, as well? 

Since `spacebar` opens the original article URL, it would be nice to have `ctrl+j` or `n` or `d` scroll down the page while reading.

**Comment by heliostatic** [comment]
I don't see Page Down listed in the shortcut settings?
I'm on version `1.10.2`, and these are the navigation and article action shortcuts I see listed:
<img width="958" alt="screenshot 2018-03-21 11 06 19" src="https://user-images.githubusercontent.com/70898/37717944-f586cb7c-2cf7-11e8-9311-826dc24bacdf.png">

**Comment by Frenzie** [comment]
D'oh! For some reason I thought this was an issue for Otter Browser. O-o

What's the use case for reimplementing Page Down in a script on a page?

**Comment by heliostatic** [comment]
Many keyboard layouts no longer expose Page Down clearly, including Apple laptops. 

Also, since the default shortcut key selection for "See on original website" hijacks default browser behavior for `spacebar` (which is page down in most browsers), it seems logical to add explicit scroll shortcuts.

Alternatively, I guess changing the default shortcut for "See on original website" would address the hijacked behavior.

**Comment by aledeg** [comment]
I think you can just change your configuration if you want your space bar back. We could also change the default value to address that issue for other users.

**Comment by Frenzie** [comment]
Yes, you can just change it to <kbd>enter</kbd> (or anything else). I've grown accustomed to space as a standard in feedreaders (from native readers like QuiteRSS to webreaders) but it's never been something I saw as anything other than an annoying hijacking in some browsers. Trying to type something while accidentally not having selected a textbox and suddenly you're somewhere completely different… It's somewhat similar with submit but that's in everything.

Anyway, I don't have a strong opinion on it. Space and enter both seem logical, and either way it's simple to change.

**Comment by ronnelson** [comment]
Changing "See on original website" from space to enter fits much better for myself at least. PageDn moves my hands off the base position, and I'm used to using space for scrolling.
But that's just my preferences...

**Comment by math-GH** [comment]
Right now we are on 1.19/1.20. Is this feature request still open?

**Comment by rcarmo** [comment]
It's 2025, and I just came here to check how to use space for paging down (and yes, I saw #342)
